Do not reindex concurrently on PG < v16

andyundso opened the pull request:
Fixes #67 

System tables in database cannot be re-indexed concurrently on PG v15 and below. This PR switches to the `reindexdb` command provided by Postgres to start the reindex operation, and depending on the version, activates `CONCURRENTLY` or not.

Without `CONCURRENTLY`, Postgres locks the database during the `INDEX` operation. I added an information about that to the `README`.

justinclift commented:
Awesome, this seems like a good approach. :smile:

The only thing that stands out as a bit unusual to me is the use of `bc` and a subshell for the value comparison:

```
$(echo "15 <= $PGTARGET" | bc)
```

Wouldn't the more standard `"$PGTARGET" -le 15` do the same thing and not need bc installed?

andyundso commented:
I am not the greatest bash programmer, but I am learning :)

removed `bc` and adjusted  the version check according to your suggestion @justinclift in the latest commit.

justinclift commented:
Cool, that looks good. :smile:
